{"uuid": "03bd80c6-ef9e-4f1b-8555-1a70a4ebd571", "counter": 7, "stdout": "\u001b[1;31mfatal: [192.168.1.1]: UNREACHABLE! => {\"changed\": false, \"msg\": \"Failed to connect to the host via ssh: ssh: connect to host 192.168.1.1 port 22: Connection refused\", \"unreachable\": true}\u001b[0m", "start_line": 5, "end_line": 6, "runner_ident": "2ecc77cd-818c-42e1-98dd-baf66686cd42", "event": "runner_on_unreachable", "pid": 9340, "created": "2022-04-22T13:46:00.880885", "parent_uuid": "9e73ed77-7fb2-050e-46de-00000000000f", "event_data": {"playbook": "deploy.yaml", "playbook_uuid": "85639527-8852-46cc-ba26-f462bb8d0147", "play": "this is a  hello world example", "play_uuid": "9e73ed77-7fb2-050e-46de-000000000007", "play_pattern": "all", "task": "Gathering Facts", "task_uuid": "9e73ed77-7fb2-050e-46de-00000000000f", "task_action": "gather_facts", "task_args": "", "task_path": "/Users/devops/data/project/python/pgoops/upload/iac/helloworld/project/deploy.yaml:2", "host": "192.168.1.1", "remote_addr": "192.168.1.1", "start": "2022-04-22T13:46:00.862054", "end": "2022-04-22T13:46:00.880794", "duration": 0.01874, "res": {"unreachable": true, "msg": "Failed to connect to the host via ssh: ssh: connect to host 192.168.1.1 port 22: Connection refused", "changed": false}, "uuid": "03bd80c6-ef9e-4f1b-8555-1a70a4ebd571"}}